app/furniture/marketplace/order.rb

```diff
@@ -11,7 +11,7 @@ class Order < Record
     has_many :ordered_products, inverse_of: :order, foreign_key: :cart_id
     has_many :products, through: :ordered_products, inverse_of: :orders
 
-    attribute :delivery_address, :string
+    has_encrypted :delivery_address
```

Comment:
What's the order of operations for everything this code change does?

Seems like in this change...
1. `:delivery_address` is being renamed to `deprecated_delivery_address`
2. data in `deprecated_delivery_address` is migrated to `delivery_address_ciphertext`
3.  data in `deprecated_delivery_address` is replaced with `nil`

And in later PR... I assume this happens?:
4. `deprecated_delivery_address` is dropped.
5. `delivery_address_ciphertext` is renamed `delivery_address` so `Order` model still works.

Reply:
Yes *except* step 5 never happens; because the `Lockbox` infers that `delivery_address_ciphertext` is the database name for the encrypted `delivery_address` attribute.